dev/date_binding.h

```diff
@@ -0,0 +1,127 @@
+#pragma once
+
+#include "functional/cxx_core_features.h"
+
+// need support for #include <compare> for advanced std::chrono members
+#if defined(SQLITE_ORM_THREE_WAY_COMPARISON_SUPPORTED)
+
+#include <chrono>
+#include <sstream>
+#include <regex>
+#include <format>
+
+inline std::chrono::sys_days Today() {
```

Comment:
1) this naming is unacceptable cause functions are named with camelCase inside lib
2) as I said before: please move this bindings from `dev` folder to example

Reply:
ok no problem...